packages/auth/blueprints/spree-ember-auth-components/files/app/components/spree-auth-orders.js

```diff
@@ -0,0 +1,21 @@
+import Ember from 'ember';
+import layout from '../templates/components/spree-auth-orders';
+/**
+ A order list for the Spree Account Route.
+
+ **To Override:** You'll need to run the components generator:
+
+ ```bash
+ ember g spree-ember-auth-components
+ ```
+
+ This will install a layout file at: `app/templates/components/spree-auth`, and
+ a component file at `app/components/spree-auth`, ready to be extended.
+
+ @class SpreeAccountOrders
+ @namespace Component
+ @extends Ember.Component
+ */
+export default Ember.Component.extend({
+  layout: layout
+});
```

Comment:
I'd rather we didn't use a component here - and instead just put the table on the account page.  The reason is that the "auth" package is concerned with Auth only, and by building out a component we're kinda muddying the concerns a bit.